client/schnorr.go

```diff
@@ -76,26 +123,18 @@ func (c *Client) open(openMsg *pb.Message) (*big.Int, error) {
 	return new(big.Int).SetBytes(bigint.X1), nil
 }
 
-func (c *Client) proofRandomData(isFirstMsg bool, a, secret *big.Int) (*pb.PedersenDecommitment, error) {
-
-	x := c.handler.schnorrProver.GetProofRandomData(secret, a)
-	b, _ := c.handler.schnorrProver.DLog.Exponentiate(a, secret)
-
+func (c *SchnorrClient) proofRandomData(isFirstMsg bool, msg *pb.Message) (*pb.PedersenDecommitment, error) {
```

Comment:
Nitpicking, but did you perhaps mean `proveRandomData` instead of `proofRandomData`? Feels like it has to be a verb instead noun, but I'm not sure.